cg/apps/orderform/excel_orderform_parser.py

```diff
@@ -22,8 +22,7 @@ class ExcelOrderformParser(OrderformParser):
     NO_VALUE: str = "no_value"
     SHEET_NAMES: List[str] = ["Orderform", "orderform", "order form"]
     VALID_ORDERFORMS: List[str] = [
-        "1508:24",  # Orderform MIP, Balsamic, sequencing only, MIP RNA
-        "1541:6",  # Orderform Externally sequenced samples
```

Comment:
this wont exist anymore?

Reply:
It's part of this PR to merge 1541 into 1508